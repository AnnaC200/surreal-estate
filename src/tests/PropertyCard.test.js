import React from "react";
import { render } from "@testing-library/react";
import PropertyCard from "../components/PropertyCard";

describe("PropertyCard", () => {
  const validProps = {
    title: "3 bed detached cottage",
    propertyType: "detached",
    bathrooms: 4,
    bedrooms: 3,
    price: 450000,
    city: "Leeds",
    email: "anna@Mcrcodes.com",
  };

  it("render", () => {
    const { asFragment } = render(
      <PropertyCard
        title={validProps.title}
        propertyType={validProps.propertyType}
        bathrooms={validProps.bathrooms}
        bedrooms={validProps.bedrooms}
        price={validProps.price}
        city={validProps.city}
        email={validProps.email}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
