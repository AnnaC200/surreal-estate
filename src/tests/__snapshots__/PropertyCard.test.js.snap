// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyCard render 1`] = `
<DocumentFragment>
  <div
    class="property-card"
  >
    <div
      class="title"
    >
      3 bed detached cottage
    </div>
    <div
      class="property-type"
    >
      detached
    </div>
    <div
      class="bathrooms"
    >
      4
    </div>
    <div
      class="bedrooms"
    >
      3
    </div>
    <div
      class="price"
    >
      450000
    </div>
    <div
      class="city"
    >
      Leeds
    </div>
    <div
      class="email"
    >
      anna@Mcrcodes.com
    </div>
  </div>
</DocumentFragment>
`;
